packages/textlint-fixer-formatter/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "paths": {},
```

Comment:
use `path` for avoiding concurrent build issue.

In following case, building is failed.

1. build kernel
2. build textlint
3. build textlint-fixer-formatter

Because, textlint depended on textlint-fixer-formatter.
2 -> 3
When `2`, `textlint-fixer-formatter.d.ts` is not exitsted

Reply:
Use the `path` setting https://github.com/textlint/textlint/pull/371#discussion_r156406004